### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Gauthier-Lhomme**
Add footer for desktop and mobile version

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Ok for me !

File: src/assets/others/IconsList.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import linkedin from "../icons/linkedin.png";
+import twitter from "../icons/twitter.png";
+
+export const ulMap = [
```
Comment (LoicBrassart) [inline src/assets/others/IconsList.jsx:4]: Rename > "IconsList"

**Review by LoicBrassart — DISMISSED** [review]
Saw no real Evil

**Review by Thoms-C — APPROVED** [review]
Clear !

File: src/components/Connexion.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+} from "../styled-components/Connexion";
+
+export default function connexion() {
+  return (
+    <Background>
+      <ContainerInscription>
+        <ContainerLogo>
+          <Logo src={logoFavart}></Logo>
+          <Slogan>
+            The best way to shape the Futur is to <Span>co create</Span> it.
```
Comment (LoicBrassart) [inline src/components/Connexion.jsx]: "Future", I think

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**